docs/Developers/Writing_Good_Commits.rst

```diff
@@ -161,6 +161,17 @@ Example 2 (implemented feature):
 
     Closes https://github.com/coala/coala/issues/2060
 
+Example 3 (fixed typo):
+
+::
+
+    configure.py: Fix docstring typo
+
+    This fixes the typo and changes
+    it from wether --> whether.
```

Comment:
Hmm... This whole text looks a bit unnecessarily comprehensive and redundant together with the shortlog